networking/configuring-networkpolicy.adoc

```diff
@@ -14,3 +14,65 @@ include::modules/nw-networkpolicy-create.adoc[leveloffset=+1]
 include::modules/nw-networkpolicy-delete.adoc[leveloffset=+1]
 
 include::modules/nw-networkpolicy-view.adoc[leveloffset=+1]
+
+include::modules/nw-networkpolicy-multitenant-isolation.adoc[leveloffset=+1]
+
+// Need to xref to a prerequisite
+[id="nw-networkpolicy-creating-default-networkpolicy-objects-for-a-new-project"]
+= Creating default NetworkPolicy objects for a new project
+
+As a cluster administrator, you can modify the global new project template to
+automatically include NetworkPolicy objects in a new project when it is created.
+
+.Prerequisites
+
+* A cluster using the OpenShift SDN network plug-in with `mode: NetworkPolicy`
+set. This mode is the default for OpenShift SDN.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* You must log in to the cluster with a user with `cluster-admin` privileges.
+* You must xref:../applications/projects/configuring-project-creation.adoc#modifying-template-for-new-projects_configuring-project-creation[create a custom template for new projects].
+
+.Procedure
+
+. Edit the default template for a new project by running the following command:
++
+----
+$ oc edit template <project_template> -n openshift-config
```

Comment:
seems the default template is not exist by default.  we need to create it via
`oc adm create-bootstrap-project-template -o yaml > template.yaml`
`oc create -f template.yaml -n openshift-config`
Then we can edit the template
@xingxingxia  please correct me if I'm wrong.

Reply:
@zhouying7780 could you help look at it when having time?

Reply:
Oh, that step is actually included as a prerequisite: _You must create a custom template for new projects._ If the user follows that link, the procedure has steps for creating a default template for projects.

Reply:
I reworked the content so the prerequisite is more visible.